lib/web_ui/lib/src/engine/initialization.dart

```diff
@@ -226,6 +226,7 @@ Future<void> initializeEngineUi() async {
   _initializationState = DebugEngineInitializationState.initializingUi;
 
   RawKeyboard.initialize(onMacOs: operatingSystem == OperatingSystem.macOs);
+  ensureImplicitViewInitialized();
```

Comment:
Will `initializeEngineUi` henceforth be used to initialize in the single-view mode?

Reply:
I haven't thought that far yet, but the way I see it is `initializeEngineUi` will be used for all modes. And instead of calling `ensureImplicitViewInitialized` unconditionally (like I'm doing here), it will be guarded by whatever conditions that dictate the need for the implicit view.

Reply:
Cool. When/if this changes, we should also update the dartdocs above.

Reply:
Also, maybe we should delay the call to `initializeEngineUi` until the first view is added? cc @ditman

Reply:
Most likely, `initializeEngineUi` will do nothing, since all it does now is `ensureFlutterViewEmbedderInitialized()`, attach a global keyboard handler (and change the state of the lifecycle of the engine initialization)! [code](https://github.com/flutter/engine/blob/d100e8912eb07bc3c36c363950561d63dc2b2e64/lib/web_ui/lib/src/engine/initialization.dart#L206-L231).

Everything it does seems that will eventually be gobbled up by `createView(viewOptions)`, right?

We can leave the method here for the "implicitView" modes, (that's where we can "createView" for the implicitView), but in multiview mode this can just let the state machine of the engine advance, and noop the rest.

Reply:
> Most likely, `initializeEngineUi` will do nothing, since all it does now is `ensureFlutterViewEmbedderInitialized()`, attach a global keyboard handler

Well, there are many things hidden behind that `ensureFlutterViewEmbedderInitialized()`. E.g. keyboard, pointers, resize, a11y, etc.

I think we can use `initializeEngineUi` for initializing the global stuff (e.g. a11y announcements).